resolwe_bio/docker_images/latex/Dockerfile

```diff
@@ -4,3 +4,7 @@ MAINTAINER Resolwe Bioinformatics authors https://github.com/genialis/resolwe-bi
 
 # Force matplotlib backend to a headless one.
 ENV MPLBACKEND Agg
+
+# Additional files:
+COPY files/multireport-template.tex /opt/resolwebio/multireport-template.tex
```

Comment:
Please choose a more explicit name. This is a general `Latex` docker image, we may want to have more templates and we want to know which template is which.